pkg/api/api.go

```diff
@@ -169,3 +173,36 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 		Status: 200,
 	})
 }
+
+// Invoke creates an event to invoke a specific function.
+func (a API) Invoke(w http.ResponseWriter, r *http.Request) {
+	// XXX: In OSS self hosting, check signing keys here.
+
+	// Get the function slug from the route parameter.   This is the function
+	// we'll invoke.  Any request is passed as the event data to the function.
+	slug := chi.URLParam(r, "slug")
```

Comment:
Could we add the ability to specify multiple functions here via `?slug[]=id&slug[]=id1`? Looks like `net` just expects [repeated values without the `[]`](https://pkg.go.dev/net/url#Values); not sure about chi.

Reply:
It's a solid idea, though for this iteration let's keep the scope small.  The current plan:

1. Release an API that allows invoking functions via the runner
2. Implement invoking functions in the SDK
3. Implement `await` as an optional parameter to await on the function response

Each of these are individual deliverables.  Once this is done, we can assess usage, gather feedback, and jam on new ideas.

Some initial thoughts:  if you're invoking multiple functions you should probably be using `inngest.send`.